dist.ini

```diff
@@ -3,7 +3,7 @@ author  = Gisle Aas <gisle@activestate.com>
 license = Perl_5
 main_module = lib/LWP/Protocol/https.pm
 copyright_holder = Gisle Aas
-copyright_year   = 2017
+copyright_year   = 1997-2019
```

Comment:
As @karenetheridge pointed out, this should probably just be 1997. I'm not sure why we ever did the year range thing (likely my fault).